richtextfx/src/main/java/org/fxmisc/richtext/EditableStyledDocument.java

```diff
@@ -19,7 +19,13 @@
 
     ReadOnlyStyledDocument<PS, S> snapshot();
 
-    EventStream<PlainTextChange> plainChanges();
+    default EventStream<PlainTextChange> plainChanges() {
+        return richChanges()
+                // map is used to prevent code repetition: StyledDocument#getText()
```

Comment:
Out of context of our previous discussion, this comment does not make much sense, does it? ;)